storagenode/pieces/readwrite_test.go

```diff
@@ -0,0 +1,85 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package pieces_test
+
+import (
+	"io"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap"
+
+	"storj.io/storj/internal/memory"
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/internal/testrand"
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/storage/filestore"
+	"storj.io/storj/storagenode/pieces"
+)
+
+func BenchmarkReadWrite(b *testing.B) {
+	ctx := testcontext.New(b)
+	defer ctx.Cleanup()
+
+	dir, err := filestore.NewDir(ctx.Dir("pieces"))
+	require.NoError(b, err)
+	blobs := filestore.New(dir)
+	defer ctx.Check(blobs.Close)
+
+	store := pieces.NewStore(zap.NewNop(), blobs)
+
+	// setup test parameters
+	const blockSize = int(256 * memory.KiB)
+	satelliteID := testrand.NodeID()
+	source := testrand.Bytes(30 * memory.MB)
+
+	testPieceID := storj.PieceID{1}
```

Comment:
can you move writing a test piece after `Write` benchmark to be clear that its used only to test `Read`?

Reply:
ah, just moved it before... but sure.